src/win/SetErrorMode.ml

```diff
@@ -0,0 +1,5 @@
+external set_error_mode: unit -> unit = "caml_SetErrorMode"
+
+let () =
```

Comment:
So this is simply being ignored, right? This does not end up in the executable?

Reply:
Yes, it is not getting built for now.